File: src/Dotnet9.VueBlog/api/models/article-report-output.ts
export interface ArticleReportOutput {
  /**
   *文章总数
   */
  articleCount: number;
  /**
   * 栏目总数
   */
  categoryCount: number;
  /**
   * 标签总数
   */
  tagCount: number;

  /**
   * 友链数量
   */
  linkCount: number;

  /**
   * 用户数量
   */
  userCount: number;
}

File: src/Dotnet9.WebAPI/Dotnet9.Application/Client/Dtos/ArticleReportOutput.cs
namespace Dotnet9.Application.Client.Dtos;

public class ArticleReportOutput
{
    /// <summary>
    /// 文章数量
    /// </summary>
    public int ArticleCount { get; set; }

    /// <summary>
    /// 标签数量
    /// </summary>
    public int TagCount { get; set; }

    /// <summary>
    /// 分类数量
    /// </summary>
    public int CategoryCount { get; set; }

    /// <summary>
    /// 专辑数量
    /// </summary>
    public int AlbumCount { get; set; }

    /// <summary>
    /// 用户量
    /// </summary>
    public int UserCount { get; set; }

    /// <summary>
    /// 友链数量
    /// </summary>
    public int LinkCount { get; set; }
}